src/pages/ReactionPage/ReactionPage.tsx

```diff
@@ -63,6 +62,7 @@ const ReactionPage = () => {
                     width={50}
                     height={50}
                   />
+                  {/* @ts-ignore there is a text for messages to band */}
```

Comment:
I think if you remove the (wrong) type [here](https://github.com/sparkletown/sparkle/pull/121/files#diff-7a3b067fec7fa24e666613c91d177246R45) you won't need the is-ignore